addons/web/static/src/core/browser/feature_detection.js

```diff
@@ -49,3 +49,7 @@ export function hasTouch() {
 export function maxTouchPoints() {
     return browser.navigator.maxTouchPoints || 1;
 }
+
+export function isDark() {
+    return getComputedStyle(document.documentElement).getPropertyValue('--o-webclient-color-scheme') === "dark";
```

Comment:
I let you to choose the cookie value or not for the detection...

Reply:
Sure, it uses cookie now ;)